readme.md

```diff
@@ -30,6 +30,11 @@ http: {
 * Indentation based syntax
 * Annotations for providing additional information
 
+The top level of a `.bru` file is an implicit Multimap. It does not require
```

Comment:
This was added because one would expect the `http:` in the example to be inside a `{}` block. It is not, which means that there is an implicit object in place.

Reply:
👍